jac-uk/admin
# #1620 - character issues google doc report

**Author checklist**


---
## What's included?
Character Issues - A new Export to Google Docs button.

Note: this issue is relative to [#1620 - character issue google doc report](https://github.com/jac-uk/digital-platform/pull/725) on digital-platform

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Preview URL: https://jac-admin-develop--pr1624-1620-character-issue-th2416y5.web.app

- View an exercise with character issues
- Click the Export to Google Docs button
- A document should be downloaded, containing an export of the character issues

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Example layout:
https://docs.google.com/document/d/12cTMJEC9j3LbR34j_4cb4iT-oHX-VthQ/edit#

Demo:

https://user-images.githubusercontent.com/79906532/171253509-1dbace62-d870-4848-a7f3-1baf2baaa22d.mov

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
